source/tutorials/longitudinal.rst

```diff
@@ -115,6 +115,50 @@ Second, control charts display the mean value of "metric" at each "state". The f
 This visualizer currently only generates control charts, which are a useful **qualitative** approach for visually identifying abnormal time points; accompanying statistical tests may be added in future releases.
 
 
+First differencing to track rate of change
+------------------------------------------
+Another way to view time series data is by assessing how the rate of change differs over time. We can do this through calculating first differences, which is the magnitude of change between successive time points. If :math:`Y_\text{t}` is the value of metric :math:`Y` at time :math:`t`, the first difference at time :math:`t`, :math:`{\Delta}Y_\text{t} = Y_\text{t} - Y_\text{t-1}`. This transformation is performed in the ``first-differences`` method in ``q2-longitudinal``.
+
+.. command-block::
+
+   qiime longitudinal first-differences \
+     --m-metadata-file ecam-sample-metadata.tsv \
+     --m-metadata-file shannon.qza \
+     --p-state-column month \
+     --p-metric shannon \
+     --o-first-differences shannon-first-differences \
+     --p-individual-id-column studyid \
+     --p-replicate-handling random
+
+This outputs a ``SampleData[FirstDifferences]`` artifact, which can then be viewed, e.g., with the ``volatility`` visualizer or analyzed with ``linear-mixed-effects`` or other methods.
+
+A similar method is ``first-distances``, which instead identifies the beta diversity distances between successive samples from the same subject. The pairwise distance between all samples can already be calculated by the ``beta`` or ``core-metrics`` methods in ``q2-diversity``, so this method simply identifies the distances between successive samples collected from the same subject and outputs this series of values as metadata that can be consumed by other methods.
+
+.. command-block::
+
+   qiime longitudinal first-distances \
+     --i-distance-matrix unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file ecam-sample-metadata.tsv \
+     --p-state-column month \
+     --o-first-distances first-distances \
+     --p-individual-id-column studyid \
+     --p-replicate-handling random
+
+This output can be used in the same way as the output of ``first-differences``. The output of ``first-distances`` is particularly empowering, though, because it allows us to analyze longitudinal changes in beta diversity using actions that cannot operate directly on a distance matrix, such as ``linear-mixed-effects``.
+
+.. command-block::
+
+   qiime longitudinal linear-mixed-effects \
+     --m-metadata-file first-distances.qza \
+     --m-metadata-file ecam-sample-metadata.tsv \
+     --p-metric Distance \
+     --p-state-column month \
+     --p-individual-id-column studyid \
+     --o-visualization distance-first-differences-LME \
```

Comment:
Can you include the `.qzv` file extension on the output? 

Also, do you think `first-distances-LME.qzv` would be a clearer output name?

Reply:
👍